# SARS-Cov-2 data source, methods, publications summary 
This repo summarize the tools used in the analysis of SARS-Cov data. Some papers are also recommended here. 


## Data Sources
### 1, GISAID: Global Initiative on Sharing All Inflenza Data: https://www.gisaid.org/
The GISAID Initiative promotes the rapid sharing of data from all influenza viruses and the coronavirus causing COVID-19. 

### 3, COG-UK Mutation Explorer: http://sars2.cvr.gla.ac.uk/cog-uk/ 
This gives the SARS-Cov data in UK, including data update, features analysis(anigenic, drug resistance, et al), and very good visulization. 

### 4, CoV-GLUE: A Web Application for Tracking SARS-CoV-2 Genomic Variation: http://cov-glue.cvr.gla.ac.uk/ 
CoV-GLUE maintains a database of mutations, insertions and deletions which have been observed in GISAID hCoV-19 sequences sampled from ongoing COVID-19 pandemic. 

### 5, Nextstrain - ncov https://github.com/nextstrain/ncov/blob/50ceffa/defaults/annotation.gff 
Genome annotation of SARS-Cov-2 in GFF format. 
There is a link avaliable for GFF format reference. https://www.ensembl.org/info/website/upload/gff.html


## Tools -- focusing on github code based on publications
### 1, Nextstrain https://github.com/nextstrain  and web 
Real-time tracking of pathogen evolution. 

There are many related tools:
#### Nextclade : Viral genome alignment, mutation calling, clade assignment, quality checks and phylogenetic placement
#### agur : Pipeline components for real-time phylodynamic analysis 
#### There are some data available. 

### 2, 



## Publications
### 1, Harvey, William T., et al. "SARS-CoV-2 variants, spike mutations and immune escape." Nature Reviews Microbiology, 2021
This review summarize literatures on mutations of SARS-CoV-2 spike protein, focusing on their impacts on antigenicity. There are also S-protein structrure to explain the mutations causing immune escape. 

### 2, 
